target/linux/x86/legacy/config-6.1

```diff
@@ -92,78 +113,172 @@ CONFIG_DRM_RADEON=y
 CONFIG_FB_SYS_FILLRECT=y
 CONFIG_FB_SYS_FOPS=y
 CONFIG_FB_SYS_IMAGEBLIT=y
 # CONFIG_FB_VESA is not set
-CONFIG_FONT_8x8=y
 CONFIG_FONT_8x16=y
+CONFIG_FONT_8x8=y
 CONFIG_FONT_SUPPORT=y
 CONFIG_FRAMEBUFFER_CONSOLE=y
 CONFIG_FRAMEBUFFER_CONSOLE_DETECT_PRIMARY=y
 # CONFIG_FRAMEBUFFER_CONSOLE_ROTATION is not set
+CONFIG_FREEZER=y
+CONFIG_FW_CACHE=y
+CONFIG_GENERIC_IRQ_EFFECTIVE_AFF_MASK=y
+CONFIG_GENERIC_IRQ_MIGRATION=y
+CONFIG_GENERIC_PENDING_IRQ=y
+CONFIG_GENERIC_PINCONF=y
 # CONFIG_GIGABYTE_WMI is not set
+CONFIG_GPIOLIB_IRQCHIP=y
+CONFIG_GPIO_ACPI=y
+CONFIG_GPIO_CDEV=y
+CONFIG_GUP_GET_PTE_LOW_HIGH=y
+CONFIG_HALTPOLL_CPUIDLE=y
 CONFIG_HDMI=y
+CONFIG_HIBERNATE_CALLBACKS=y
 CONFIG_HID_BATTERY_STRENGTH=y
+CONFIG_HID_GENERIC=y
+CONFIG_HID_HYPERV_MOUSE=y
 # CONFIG_HIGHMEM4G is not set
+CONFIG_HOTPLUG_CPU=y
+CONFIG_HOTPLUG_PCI=y
+CONFIG_HOTPLUG_PCI_ACPI=y
+# CONFIG_HOTPLUG_PCI_ACPI_IBM is not set
+# CONFIG_HOTPLUG_PCI_COMPAQ is not set
+# CONFIG_HOTPLUG_PCI_CPCI is not set
+# CONFIG_HOTPLUG_PCI_IBM is not set
+CONFIG_HOTPLUG_PCI_PCIE=y
+# CONFIG_HOTPLUG_PCI_SHPC is not set
+CONFIG_HOTPLUG_SMT=y
 CONFIG_HPET=y
 CONFIG_HPET_MMAP=y
 # CONFIG_HP_ACCEL is not set
 # CONFIG_HUAWEI_WMI is not set
+CONFIG_HVC_DRIVER=y
+CONFIG_HVC_IRQ=y
+CONFIG_HVC_XEN=y
+CONFIG_HVC_XEN_FRONTEND=y
 CONFIG_HWMON=y
+CONFIG_HWMON_VID=y
+CONFIG_HW_RANDOM_VIRTIO=y
+CONFIG_HYPERV=y
+CONFIG_HYPERVISOR_GUEST=y
+CONFIG_HYPERV_BALLOON=y
+CONFIG_HYPERV_KEYBOARD=y
+CONFIG_HYPERV_NET=y
+CONFIG_HYPERV_STORAGE=y
+# CONFIG_HYPERV_TESTING is not set
+CONFIG_HYPERV_TIMER=y
+CONFIG_HYPERV_UTILS=y
 CONFIG_I2C=y
 CONFIG_I2C_ALGOBIT=y
 # CONFIG_I2C_AMD_MP2 is not set
 CONFIG_I2C_BOARDINFO=y
 # CONFIG_I2C_HID_ACPI is not set
 # CONFIG_I2C_MULTI_INSTANTIATE is not set
 # CONFIG_I8K is not set
+# CONFIG_IMA_SECURE_AND_OR_TRUSTED_BOOT is not set
 CONFIG_INPUT_MOUSE=y
 CONFIG_INPUT_MOUSEDEV=y
 CONFIG_INPUT_MOUSEDEV_PSAUX=y
 CONFIG_INPUT_MOUSEDEV_SCREEN_X=1024
 CONFIG_INPUT_MOUSEDEV_SCREEN_Y=768
+CONFIG_INPUT_XEN_KBDDEV_FRONTEND=y
 CONFIG_INTEL_GTT=y
 CONFIG_INTEL_IDLE=y
 # CONFIG_INTEL_IPS is not set
 # CONFIG_INTEL_MEI_HDCP is not set
 # CONFIG_INTEL_MEI_PXP is not set
 # CONFIG_INTEL_MENLOW is not set
+CONFIG_INTEL_PCH_THERMAL=y
 # CONFIG_INTEL_SAR_INT1092 is not set
 # CONFIG_INTEL_SCU_PLATFORM is not set
+CONFIG_INTEL_SOC_DTS_IOSF_CORE=y
 # CONFIG_INTEL_SOC_DTS_THERMAL is not set
+CONFIG_INTEL_SOC_DTS_THERMAL=y
 # CONFIG_INTEL_WMI_SBL_FW_UPDATE is not set
 # CONFIG_INTEL_WMI_THUNDERBOLT is not set
 CONFIG_INTERVAL_TREE=y
 CONFIG_IOSF_MBI=y
 # CONFIG_IOSF_MBI_DEBUG is not set
+CONFIG_IRQ_BYPASS_MANAGER=y
 CONFIG_ISA=y
 CONFIG_ISAPNP=y
 CONFIG_ISA_BUS_API=y
 # CONFIG_ISCSI_IBFT is not set
 CONFIG_ISO9660_FS=y
 # CONFIG_JOLIET is not set
 CONFIG_KCMP=y
+CONFIG_KVM=y
+CONFIG_KVM_AMD=y
+CONFIG_KVM_ASYNC_PF=y
+CONFIG_KVM_GENERIC_DIRTYLOG_READ_PROTECT=y
+CONFIG_KVM_GUEST=y
+CONFIG_KVM_INTEL=y
+CONFIG_KVM_MMIO=y
+CONFIG_KVM_VFIO=y
+# CONFIG_KVM_XEN is not set
+CONFIG_KVM_XFER_TO_GUEST_WORK=y
```

Comment:
I'm unsure kernel-VM was back-ported to function on `legacy`-class hardware.  Yet this had been enabled in `generic`.

Reply:
Maybe someone wants to use a 32 bit OpenWrt on a modern CPU in KVM.

Reply:
The danger is it could bloat `legacy` to the point it was too big for actual legacy hardware.  Between legacy _hardware_ and 32-bit KVM, legacy hardware seems the more valuable target.